backend/controllers/adminControllers.js

```
@@ -0,0 +1,192 @@
+const { User } = require('../models/userSchemas');
+const { VendorRequest } = require("../models/vendorRequestSchemas");
+const mongoose = require('mongoose');
+const jwt = require("jsonwebtoken");
+
+
+const getUsers = async (req, res) => {
+  try {
+    const page = parseInt(req.query.page) || 1;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)